components/Workspaces/WorkspaceLayout.tsx

```diff
@@ -36,34 +37,41 @@ export const WorkspaceLayout = ({ workspaceId, banner, children }: WorkspaceLayo
   );
 
   return (
-    <div
-      className="grid"
-      style={{
-        "--top-nav-height": "3.3rem",
-        gridTemplateRows: "var(--top-nav-height) auto 1fr",
-        minHeight: "calc(100vh - var(--top-nav-height))",
-      }}
-    >
-      <div>
-        <TopNav />
-        <ClientOnly>
-          <span ref={sidebarRef}>
-            <AppSideBar workspaceId={workspaceId} hideSidebar={hideSidebar} sidebarCollapsed={showingSidebar} />
-          </span>
-          {!showingSidebar && (
-            <button
-              onClick={() => setShowingSidebar(true)}
-              className="fixed z-50 border-8 border-solid border-black inset-x-0 top-16 w-fit bg-white rounded-r-md shadow-lg p-2"
-            >
-              <LuArrowRightToLine className="w-4 h-4 text-gray-500" />
-            </button>
-          )}
-        </ClientOnly>
+    <>
+      <div
+        className="grid"
+        style={{
+          "--top-nav-height": "3.3rem",
+          gridTemplateRows: "var(--top-nav-height) auto 1fr",
+          minHeight: "calc(100vh - var(--top-nav-height))",
+        }}
+      >
+        <div>
+          <TopNav />
+          <ClientOnly>
+            <span ref={sidebarRef}>
+              <AppSideBar workspaceId={workspaceId} hideSidebar={hideSidebar} sidebarCollapsed={showingSidebar} />
+            </span>
+            {!showingSidebar && (
+              <button
+                onClick={() => setShowingSidebar(true)}
+                className="fixed z-50 border-8 border-solid border-black inset-x-0 top-16 w-fit bg-white rounded-r-md shadow-lg p-2"
+              >
+                <LuArrowRightToLine className="w-4 h-4 text-gray-500" />
+              </button>
+            )}
+          </ClientOnly>
+        </div>
+        <div className="relative flex flex-col items-center grow pt-8 md:pt-14 lg:pt-20">
+          {banner}
+          <div className="px-1 sm:px-2 md:px-4 xl:px-16 container w-full min-h-[100px] pb-32">{children}</div>
```

Comment:
I added padding at the bottom so that you can see all the content with the fixed footer and also, it's just nice to have some empty space at the end when scrolling.